tests/test_clients.py

```diff
@@ -49,11 +49,50 @@ def test_close_without_clients(hlwm):
 
 
 @pytest.mark.parametrize("running_clients_num", [1, 2, 3, 4])
-def test_close(hlwm, running_clients_num):
+def test_close_for_multiple_clients(hlwm, running_clients_num):
```

Comment:
This test and some others related to the "close" command do not assert anything.
Is this inteded? How is it tested, that the clients are actually closed?

Reply:
I also can't remember what's the purpose of this test. It's inconvenient to wait for a client to close if we don't have it's process handle. Maybe this test case just checked that hlwm does not crash. Maybe I should just extend `test_close_focused_client` by other unfocused clients to verify that 'close' only closes the focused one.

Reply:
I've removed this incomplete test and extended the other test where it is actually verified that the focused client closes.

Reply:
Thanks for updating the test!
Should we add checks for the client to be closed in the unfocused and unmanaged case as well?

Reply:
In the unfocused and unmanaged case, it is already checked that it is closed (if something is closed, then there is `proc.wait(10)` in the test.)

Reply:
Aaah, yes, you're right. I was looking for asserts, but the proc.wait() is the assert in that case